src/app/organization/offices/create-offices/create-offices.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="container">
+  <mat-card>
+    <form [formGroup]="newOfficeForm" (ngSubmit)="submit()">
+      <mat-card-content>
+        <div fxLayout="column" fxLayoutAlign="center">
+          <mat-form-field fxFlex="100%">
+            <mat-label>Office</mat-label>
+            <input required matInput formControlName="name" />
+            <mat-error *ngIf="newOfficeForm.controls.name.hasError('required')">
+              Office is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field fxFlex="100%">
+            <mat-label>Parent Office</mat-label>
+            <mat-select required formControlName="parentId">
+              <mat-option *ngFor="let office of officeData" [value]="office.id">
+                {{ office.name }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="newOfficeForm.controls.parentId.hasError('required')">
+              Parent Office is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field fxFlex="100%">
+            <mat-label>Opened On</mat-label>
+            <input matInput [min]="minDate" [max]="maxDate" formControlName="openingDate" required [matDatepicker]="openedOnDatePicker">
+            <mat-datepicker-toggle matSuffix [for]="openedOnDatePicker"></mat-datepicker-toggle>
+            <mat-datepicker #openedOnDatePicker></mat-datepicker>
+            <mat-error *ngIf="newOfficeForm.controls.openingDate.hasError('required')">
+              Opening Date is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+          <mat-form-field fxFlex="100%">
```

Comment:
Ensure proper indentation/spacing in the entire file. Have a look at the other component for reference.